app/lib/router/shell_routers/home_shell_router.dart

```diff
@@ -0,0 +1,293 @@
+import 'package:acter/common/utils/routes.dart';
+import 'package:acter/features/home/pages/dashboard.dart';
+import 'package:acter/features/profile/pages/my_profile_page.dart';
+import 'package:acter/features/settings/pages/blocked_users.dart';
+import 'package:acter/features/settings/pages/email_addresses.dart';
+import 'package:acter/features/settings/pages/index_page.dart';
+import 'package:acter/features/settings/pages/info_page.dart';
+import 'package:acter/features/settings/pages/labs_page.dart';
+import 'package:acter/features/settings/pages/licenses_page.dart';
+import 'package:acter/features/settings/pages/notifications_page.dart';
+import 'package:acter/features/settings/pages/sessions_page.dart';
+import 'package:acter/features/space/pages/chats_page.dart';
+import 'package:acter/features/space/pages/events_page.dart';
+import 'package:acter/features/space/pages/members_page.dart';
+import 'package:acter/features/space/pages/overview_page.dart';
+import 'package:acter/features/space/pages/pins_page.dart';
+import 'package:acter/features/space/pages/related_spaces_page.dart';
+import 'package:acter/features/space/pages/tasks_page.dart';
+import 'package:acter/features/space/providers/space_navbar_provider.dart';
+import 'package:acter/features/space/settings/pages/apps_settings_page.dart';
+import 'package:acter/features/space/settings/pages/index_page.dart';
+import 'package:acter/features/spaces/pages/join_space.dart';
+import 'package:acter/features/spaces/pages/spaces_page.dart';
+import 'package:acter/router/router.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:go_router/go_router.dart';
+
+List<RouteBase> makeHomeShellRoutes(ref) {
```

Comment:
Great work! I like how the shell routes have been kept separately from general routes as they keep and would getting complex and makes easier for future refactors.

Reply:
Yes, it is getting complex if we are using all the routers in single file. So separated each routers based on their use-case. Easier to maintain in future as well. :)